src/cmd-generate-release-meta

```diff
@@ -37,10 +41,12 @@ def get_extension(path, modifier, arch):
 parser = ArgumentParser()
 parser.add_argument("--workdir", help="cosa workdir")
 parser.add_argument("--build-id", help="build id")
+parser.add_argument("--distro", help="Distro selects stream defaults such as baseurl and format", choices=['fcos', 'rhcos'])
 parser.add_argument("--stream-name", help="Override the stream ID (default is derived from coreos-assembler)")
-parser.add_argument("--stream-baseurl", help="Prefix URL for stream content", default=FCOS_STREAM_ENDPOINT)
+parser.add_argument("--stream-baseurl", help="Override prefix URL for stream content")
```

Comment:
But OTOH dropping the `default=` on this one *would* break FCOS. Hmm, let's just have it key off of `--distro` if unspecified and then we definitely do follow up this PR with one for the pipeline?

Reply:
Ah yep, good catch!   I actually had a "dispatch on distro" code but then removed it, this change was an artifact of that.  For now to keep things even clearer and minimize distro-dispatching I just reverted it.